### DISCUSSION TITLE
Error on signup

### DISCUSSION THREAD

**Discussion by alfredosalzillo** [supabase/supabase#490] [Questions]
I'm getting this error when using the signup

```json
{"code":500,"msg":"Database error saving new user","error_id":"2d159958-6560-4c7c-a212-4e1c14127551"}
```

How can I read the error  with id `2d159958-6560-4c7c-a212-4e1c14127551`?

**Comment by kiwicopple** [comment]
Hey @alfredosalzillo we having added detailed logging yet unfortunately. Did you do anything with your `auth` schema? Perhaps removed a column? 

You can run this and drop the output here:

```sql
SELECT *
 FROM information_schema.columns
 WHERE table_schema = 'auth' AND table_name = 'users';
```

You should have the following

column_name | data_type
-- | --
instance_id | uuid
id | uuid
aud | character varying
role | character varying
email | character varying
encrypted_password | character varying
confirmed_at | timestamp with time zone
invited_at | timestamp with time zone
confirmation_token | character varying
confirmation_sent_at | timestamp with time zone
recovery_token | character varying
recovery_sent_at | timestamp with time zone
email_change_token | character varying
email_change | character varying
email_change_sent_at | timestamp with time zone
last_sign_in_at | timestamp with time zone
raw_app_meta_data | jsonb
raw_user_meta_data | jsonb
is_super_admin | boolean
created_at | timestamp with time zone
updated_at | timestamp with time zone

**Reply by alfredosalzillo** [reply]
It looks correct
```
column_name;data_type
instance_id;uuid
id;uuid
aud;character varying
role;character varying
email;character varying
encrypted_password;character varying
confirmed_at;timestamp with time zone
invited_at;timestamp with time zone
confirmation_token;character varying
confirmation_sent_at;timestamp with time zone
recovery_token;character varying
recovery_sent_at;timestamp with time zone
email_change_token;character varying
email_change;character varying
email_change_sent_at;timestamp with time zone
last_sign_in_at;timestamp with time zone
raw_app_meta_data;jsonb
raw_user_meta_data;jsonb
is_super_admin;boolean
created_at;timestamp with time zone
updated_at;timestamp with time zone
```

**Reply by kiwicopple** [reply]
OK, can you please provide some steps to replicate? At the moment it's hard for me to know if it's a problem with your schema or your code.
